helm/reana/templates/reana-server.yaml

```diff
@@ -77,15 +78,26 @@ spec:
             value: {{ .Values.quota.default_disk_limit | default 0 | quote }}
           - name: REANA_KUBERNETES_JOBS_MEMORY_LIMIT
             value: {{ .Values.kubernetes_jobs_memory_limit | default "4Gi" }}
+          {{- $pathsList := list }}
+          {{- range $workspace_paths := .Values.workspaces.paths }}
+          {{- $workspace_path := split ":" $workspace_paths }}
+          {{- $pathsList = append $pathsList $workspace_path._1}}
+          {{- end }}
           - name: WORKSPACE_PATHS
-            value: {{ join "," .Values.workspaces.paths }}
-          {{- if not (eq (index .Values.workspaces.paths 0) "/var/reana")}}
+            value: {{ join "," $pathsList }}
```

Comment:
Syntax of defining  `WORKSPACE_PATHS` and `DEFAULT_WORKSPACE_PATH` seems to be really complicated when we have to exclude `"/var/reana"` and pick the first list item. To do it in Helm is a bit complex, hard to read and maintain. So I would propose to simply expose `WORKSPACE_PATHS` through Helm :
```
- name: WORKSPACE_PATHS
  value: {{ join "," .Values.workspaces.paths }}
  ```
and handle all the complexity in `reana-commons` while defining these envars. WDYT?